src/frontend/frontend/views/auth_views.py

```
@@ -70,26 +71,28 @@ def get(self):
         username = request.form.get("username")
         password = request.form.get("password")
 
         if not username or not password:
-            return render_login_page(login_error="Username and password are required"), 400
+            return render_login_page(login_error=gettext("Username and password are required")), 400
 
         try:
             core_response = CoreApi().login(username, password)
         except HTTPException:
             raise
         except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)